llvm/test/CodeGen/AIE/aie2/schedule/resource/dc_wm.mir

```diff
@@ -165,3 +166,37 @@ body:             |
   $dc1 = MOVA_lda_cg 2
   $dc1 = MOVA_lda_cg 2
 ...
+
+---
+name:            load_to_M_and_move_R_to_DC
+alignment:       16
+body:             |
+  bb.0.entry:
+    ; CHECK-LABEL: name: load_to_M_and_move_R_to_DC
+    ; CHECK: BUNDLE implicit-def $m0, implicit-def $dc0, implicit killed $p7, implicit killed $r0 {
+    ; CHECK-NEXT:   $m0 = LDA_dms_lda_idx_imm killed $p7, 0
+    ; CHECK-NEXT:   $dc0 = MOV_mv_scl killed $r0
+    ; CHECK-NEXT: }
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    ; CHECK-NEXT: NOP
+    $m0 = LDA_dms_lda_idx_imm killed $p7, 0
+    $dc0 = MOV_mv_scl $r0
```

Comment:
I don't think the test proves something, the ports are used at very different cycles, so there will never be a conflict. Maybe have 7 `$dc0 = MOV_mv_scl $r0` in total to be sure one of them uses a write port at the same time as LDA?